flancer64/magento2
# Remove existing rates and compose cross rates

### Description (*)
We should remove old rates we does not use and we need to calculate cross-rates for all used currencies because sales order grid displays wrong currency labels for missed rates (`Grand Total (Purchased)` column).

### Manual testing scenarios (*)
Save these currency rates:
![image](https://user-images.githubusercontent.com/5052385/54624909-f647ae80-4a76-11e9-98ba-87e549bd415c.png)

then check `directory_currency_rate` table:
```
currency_from|currency_to|rate          
-------------|-----------|--------------
CNY          |EUR        |0.131147540984
CNY          |USD        |0.149031296572
EUR          |CNY        |7.625000000000
EUR          |USD        |1.136363636364
USD          |CNY        |6.710000000000
USD          |EUR        |0.880000000000
```

### Contribution checklist (*)
